mmcv/runner/checkpoint.py

```diff
@@ -220,7 +227,10 @@ def _register_scheme(cls, prefixes, loader, force=False):
             sorted(cls._schemes.items(), key=lambda t: t[0], reverse=True))
 
     @classmethod
-    def register_scheme(cls, prefixes, loader=None, force=False):
+    def register_scheme(cls,
+                        prefixes: Union[str, List[str], Tuple[str, ...]],
```

Comment:
Suggest to use `Sequence[str]` to replace `List[str], Tuple[str, ...]`, and modify the docstring with prefixes (str, Sequence[str])